tasks/libs/pipeline_notifications.py

```diff
@@ -56,13 +58,16 @@ def check_for_missing_owners_slack_and_jira(print_missing_teams=True, owners_fil
     return error
 
 
-def get_failed_tests(project_name, job, owners_file=".github/CODEOWNERS"):
-    gitlab = Gitlab(project_name=project_name, api_token=get_gitlab_token())
+def get_failed_tests(project_name, job: ProjectJob, owners_file=".github/CODEOWNERS"):
+    repo = get_gitlab_repo(project_name)
     owners = read_owners(owners_file)
-    test_output = gitlab.artifact(job["id"], "test_output.json", ignore_not_found=True)
+    try:
+        test_output = str(repo.jobs.get(job.id, lazy=True).artifact('test_output.json'), 'utf-8')
```

Comment:
Not related to your change, but I'm wondering why we don't `json.loads` the `test_output.json` right away but line by line below.

Reply:
This file is not really a json file but contains json for each line, e.g. :
```json
{"hello": "world"}
{"cats": [...]}
```